app/controllers/api/users.php

```diff
@@ -530,6 +530,10 @@ function createUser(string $hash, mixed $hashOptions, string $userId, ?string $e
         $memberships = array_map(function ($membership) use ($dbForProject, $user) {
             $team = $dbForProject->getDocument('teams', $membership->getAttribute('teamId'));
 
+            if ($team->isEmpty()) {
+                throw new Exception(Exception::TEAM_NOT_FOUND);
+            }
+
```

Comment:
Hmm...this would cause an error so when you browse to the page, nothing would show even if they had other teams they were in, right?